### ISSUE TITLE
CMake and CI

### ISSUE THREAD

**Issue by janba** [glfw/glfw#1832]
Dear GLFW developers,

TL;DR: On windows, CMake appears to be unable to locate GLFW3 even if it is installed in a proper way.

To be specific, I am trying to [automate](https://github.com/janba/GEL/blob/janba-patch-1/.github/workflows/cmake.yml) building of the [GEL library](https://github.com/janba/GEL) using GitHub Actions. On Windows, the following command is issued to install GLFW3:

```vcpkg install glfw3```

This works fine and installs GLFW 3.3.2 (apparently from source). The output says that

```
The package glfw3:x86-windows provides CMake targets:
  find_package(glfw3 CONFIG REQUIRED)
  target_link_libraries(main PRIVATE glfw)
```
but later when CMake runs, I get the error that 

```Could not find a package configuration file provided by "glfw3" with any of```

There is [some more text](https://github.com/janba/GEL/runs/1678146291?check_suite_focus=true) that I suspect is very familiar to many people. Now, I could obviously fix this easily enough by manually setting the path to headers and library, but I am trying to automate the compilation and that would probably be a brittle solution. Besides, it works fine on MacOS and Linux, just not on Windows.

What gives? 

I believe this might have been discussed previously. Is the problem hard to fix such that we can use this excellent library on Windows without pain? If so why? 

Thanks a lot for any comments!

Andreas

**Comment by janba** [comment]
Ok - based on another thread, I decided that the best way of making CMake aware of the existence of another project is to use FetchContent, so I will use that going forward. At least, it now compiles on all three platforms with GitHub Actions.
